backend/pyproject.toml

```diff
@@ -48,10 +44,10 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.11"
 fastapi = { extras = ["all"], version = "^0.104.1" }
+githubkit = { version = "^0.11.0a4", allow-prereleases = true }
```

Comment:
why allow prereleases here specifically? I'm curious

Reply:
If I remember correctly it was to get the latest version of `githubkit` that was compatible for `Pydantic V2`